examples/camera/2d_screen_shake.rs

```diff
@@ -6,175 +6,102 @@
         Transform::from_xyz(0., 0., 2.),
     ));
 
     commands.spawn((
-        Mesh2d(meshes.add(Rectangle::new(50.0, 50.0))), // Rectangle size (width, height)
-        MeshMaterial2d(materials.add(Color::srgb(0.85, 0.0, 0.2))), // RGB values must be in range 0.0 to 1.0
+        Mesh2d(meshes.add(Rectangle::new(50.0, 50.0))),
+        MeshMaterial2d(materials.add(Color::srgb(0.85, 0.0, 0.2))),
         Transform::from_xyz(-450.0, 200.0, 2.),
     ));
 
     commands.spawn((
-        Mesh2d(meshes.add(Rectangle::new(70.0, 50.0))), // Rectangle size (width, height)
-        MeshMaterial2d(materials.add(Color::srgb(0.5, 0.8, 0.2))), // RGB values must be in range 0.0 to 1.0
+        Mesh2d(meshes.add(Rectangle::new(70.0, 50.0))),
+        MeshMaterial2d(materials.add(Color::srgb(0.5, 0.8, 0.2))),
         Transform::from_xyz(450.0, -150.0, 2.),
     ));
-    commands.init_resource::<ScreenShake>();
-}
-
-fn setup_instructions(mut commands: Commands) {
-    commands.spawn((
-        Text::new("Hold space to trigger a screen shake"),
-        Node {
-            position_type: PositionType::Absolute,
-            bottom: Val::Px(12.0),
-            left: Val::Px(12.0),
-            ..default()
-        },
-    ));
 }
 
-fn setup_camera(mut commands: Commands) {
-    commands.spawn((
-        Camera2d,
-        Camera {
-            sub_camera_view: Some(SubCameraView {
-                full_size: UVec2::new(1000, 700),
-                offset: Vec2::new(0.0, 0.0),
-                size: UVec2::new(1000, 700),
-            }),
-            order: 1,
-            ..default()
-        },
-    ));
-}
-
-#[derive(Resource, Clone)]
-struct ScreenShake {
-    max_angle: f32,
-    max_offset: f32,
-    trauma: f32,
-    latest_position: Option<Vec2>,
-}
-
-impl Default for ScreenShake {
-    fn default() -> Self {
-        Self {
-            max_angle: 0.0,
-            max_offset: 0.0,
-            trauma: 0.0,
-            latest_position: Some(Vec2::default()),
-        }
-    }
-}
-
-impl ScreenShake {
-    fn start_shake(&mut self, max_angle: f32, max_offset: f32, trauma: f32, final_position: Vec2) {
-        self.max_angle = max_angle;
-        self.max_offset = max_offset;
-        self.trauma = trauma.clamp(0.0, 1.0);
-        self.latest_position = Some(final_position);
-    }
-}
-
-fn trigger_shake_on_space(
+fn trigger_shake(
     time: Res<Time>,
     keyboard_input: Res<ButtonInput<KeyCode>>,
     mut screen_shake: ResMut<ScreenShake>,
 ) {
     if keyboard_input.pressed(KeyCode::Space) {
-        let screen_shake_clone = screen_shake.clone();
-        screen_shake.start_shake(
-            MAX_ANGLE,
-            MAX_OFFSET,
-            screen_shake_clone.trauma + TRAUMA_INCREMENT * time.delta_secs(),
-            Vec2 { x: 0.0, y: 0.0 },
-        ); // final_position should be your current player position
+        screen_shake.amount += AMOUNT_INCREMENT * time.delta_secs();
```

Comment:
Previously, `amount` (`trauma`) would be clamped to `0.0..1.0`, which is probably something we want to keep.